blueprints/addon/files/README.md

```diff
@@ -3,6 +3,13 @@
 
 [Short description of the addon.]
 
+
+Compatibility
+------------------------------------------------------------------------------
+
+* Ember.js v2.18 or above
```

Comment:
I just remembered that we now put addons on `ember-cli-babel@7`, which means that they need "Ember CLI 2.13 or above" too. We should probably add that too.

Reply:
We also only test LTS by default, so while Ember v3.1 is technically above Ember.js v2.18, it isn't really compatible because it isn't tested. Not sure if we should indicate this or not though.

Reply:
the big question here is if addons should state that they only support the latest two LTS releases. that would mean dropping support for older Ember releases could be done without bumping major versions, which I'm personally not a fan of.